### PR TITLE
Support multiple projects

### PR DESCRIPTION
**Pull request by jmatraszek**
1. Will require to drop the db before running migrations (`the-big-red-button` will not work at that time (as it expects different db structure) so either use the previously compiled version or drop directly  from `psql` by `TRUNCATE commits; TRUNCATE releases;`).
2. This does no include any support for frontend. Frontend will work as it used to work, but it will assume all commits are Rust's.
3. Added `contributors` table, which is still not used (to decide if we want to use it in the future, but probably yes). But since the project was renamed maybe we should stick with `authors`, which sound more git-ish.
4. Tested it with a couple of small projects and everything worked well, but more testing is MUCH APPRECIATED.
5. The implementation is quite naive in some places, but this can be improved in the future (didn't want to wait with this pull request).

### CODE REVIEW

File: migrations/20170125121136_create_contributors/up.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE contributors (
+    id SERIAL NOT NULL,
+    name VARCHAR,
+    email VARCHAR
```
Comment (steveklabnik) [inline migrations/20170125121136_create_contributors/up.sql]: should these two also be `NOT NULL`?

File: src/lib.rs [hunk 2]
```diff
@@ -119,7 +135,9 @@ pub fn inaccurate_sort(strings: &mut Vec<String>) {
     strings.sort_by(|a, b| str_cmp(&a, &b));
 }
 
-pub fn assign_commits(log: &slog::Logger, release_name: &str, previous_release: &str, path: &str) {
+pub fn assign_commits(log: &slog::Logger, release_name: &str, previous_release: &str, release_project_id: i32, path: &str) {
+    // Could take the connection as a parameter, as problably
+    // it's already established somewhere...
```
Comment (steveklabnik) [inline src/lib.rs]: yeah; i'm totally okay with doing this this way for now; i want to clean up all the connection stuff. we'll get there.

File: migrations/20170125121136_create_contributors/up.sql (hunk 1 above)
Comment (jmatraszek) [inline migrations/20170125121136_create_contributors/up.sql]: Fixed!

**Comment by jmatraszek** [comment]
@steveklabnik Changed the migration & merged current master, so it does not have merge conflicts.

**Comment by steveklabnik** [comment]
Thanks a ton! Not going to merge just yet as I'd like to test locally first; but this looks awesome ❤️

**Comment by steveklabnik** [comment]
I am in the process of deploying this right now, thanks so much @jmatraszek !

**Pull request merged**